lib/train/extras/file_windows.rb

```diff
@@ -51,6 +51,26 @@ def mounted
       nil
     end
 
+    # PS C:\> Get-Acl .\test | select owner,group,audit | ConvertTo-Json
+    # {
+    #     "Owner":  "BUILTIN\\Administrators",
+    #     "Group":  "DEFAULT-CHEF-WI\\None",
+    # }
+    def owner
+      cmd = @backend.run_command(
+        "Get-Acl '#{@spath}' | select owner, group | ConvertTo-Json"
```

Comment:
If you just want the owner, you can just return stdout from `Get-Acl '#{@spath}' | select -expand Owner`